### DISCUSSION TITLE
What is the IOU interval for AP for bbox eval?

### DISCUSSION THREAD

**Discussion by pjoyce1995** [open-mmlab/mmdetection#8959] [Q&A]
I am referring to the AP listed in the table broken down by category.

I cannot find the appropriate directory to find this myself. It seems that the mean_ap.py code in mmdet.core.evaluation isn't used when running ./tools/test.py

The mean_ap.py calculates AP by the '11 points' as the average between IOU of 0 to 1, but other sources say that COCO uses IOU between 0.5 and 0.95.

**Comment by zachcoleman** [comment]
Most evaluation is driven by the dataset you are evaluating against and in the corresponding code. For COCO you are correct:
https://github.com/open-mmlab/mmdetection/blob/9d3e162459590eee4cfc891218dfbb5878378842/mmdet/datasets/coco.py#L616-L620
For the mmdet CustomDataset it is here:
https://github.com/open-mmlab/mmdetection/blob/9d3e162459590eee4cfc891218dfbb5878378842/mmdet/datasets/custom.py#L327

It is whatever you configure with your test dataset for the evaluate function to use.
